client/app/locales/locale-de.json

```diff
@@ -0,0 +1,10 @@
+{
+  "LOGIN": {
+    "TITLE": "Login",
+    "EMAIL": "E-Mail",
+    "PASSWORD": "Passwort",
+    "WRONG": "Die Anmeldedaten sind fehlerhaft. Bitte überprüfe sie sorgfältig.",
+    "SUBMIT": "Login",
+    "SIGNUP": "Neuanmeldung"
+  }
+}
```

Comment:
Not sure which you thought is the better solution. It's not obvious to me either way.